zeroex/orderwatch/order_watcher.go

```diff
@@ -288,6 +292,20 @@ func (w *Watcher) mainLoop(ctx context.Context) error {
 	}
 }
 
+func drainBlockEventsChan(blockEventsChan chan []*blockwatch.Event) []*blockwatch.Event {
+	allEvents := []*blockwatch.Event{}
+L:
+	for {
+		select {
+		case moreEvents := <-blockEventsChan:
+			allEvents = append(allEvents, moreEvents...)
+		default:
+			break L
```

Comment:
Why is the label needed here? Does break only break out of the `select`?

Reply:
Yes, without this label, `break` would only break out of `select`. @fabio maybe we could rename the label to `Loop` or `OuterLoop`?